player/command.c

```diff
@@ -6496,6 +6496,16 @@ static void cmd_dump_cache_ab(void *p)
                  cmd->args[0].v.s);
 }
 
+static void cmd_begin_vo_dragging(void *p)
+{
+    struct mp_cmd_ctx *cmd = p;
+    struct MPContext *mpctx = cmd->mpctx;
+    struct vo *vo = mpctx->video_out;
+
+    if (vo)
+        vo_control(vo, VOCTRL_BEGIN_DRAGGING, NULL);
```

Comment:
Note that VOCTRL are delayed to next render loop, so keep that in mind, because it may add unwanted latency on drag begin. Or maybe it is not a problem, but VOCTRL system in general is not that great for doing stuff "now" it is more for "put request on queue and do it next time"